app/scripts/directives/metrics.js

```diff
@@ -351,6 +373,9 @@ angular.module('openshiftConsole')
             });
           } else {
             sparklineByMetric[chartId].load(sparklineData);
+            sparklineByMetric[chartId].axis.max({
+              y: yAxisMax
```

Comment:
[merge]

On Thu, Jul 7, 2016 at 2:43 PM, Sam Padgett <notifications@github.com>
wrote:

> In app/scripts/directives/metrics.js
> <https://github.com/openshift/origin-web-console/pull/139#discussion_r69963312>
> :
>
> > @@ -351,6 +373,9 @@ angular.module('openshiftConsole')
> >              });
> >            } else {
> >              sparklineByMetric[chartId].load(sparklineData);
> > +            sparklineByMetric[chartId].axis.max({
> > +              y: yAxisMax
>
> Right, the behavior will be as before for larger values.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/openshift/origin-web-console/pull/139/files/481f633c929448229c91b85b98b7ace91eb4ef5f#r69963312>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/ABZk7YUQQYTP8KkQt_Cm_U3nhopo5KBQks5qTUjGgaJpZM4JHWkx>
> .
>